lib/vulneruby/trigger.rb

```diff
@@ -7,5 +7,6 @@ module Trigger
   end
 end
 require_relative './trigger/cmd_injection'
+require_relative './trigger/path_traversal'
```

Comment:
do we need to require `reflection_injection` here?

Reply:
We should yep